onnxscript/ir/_core.py

```diff
@@ -470,11 +470,14 @@ class ExternalTensor(TensorBase, _protocols.TensorProtocol):  # pylint: disable=
     To obtain an array, call :meth:`numpy`. To obtain the bytes,
     call :meth:`tobytes`.
 
-    The :attr:`path` can be a relative path or an absolute path.
-    Serializers should handle the path correctly to conform with the ONNX spec.
+    The :attr:`location` must be a relative path conforming to the ONNX
+    specification. Given the correct :attr:`base_dir`, the :attr:`path` is computed
+    to be the full path to the data file. Users should expect that the :attr:`path`
+    always leads to the correct file.
 
     Attributes:
-        path: The path to the data file. This can be a relative path or an absolute path.
+        path: The path to the data file.
```

Comment:
This line no longer needed?

Reply:
It is still relevant because path is one of the attributes of this class (it is computed)